.travis.yml

```diff
@@ -3,11 +3,6 @@ language: perl
 matrix:
     fast_finish: true
 perl:
-   - '5.24'
-   - '5.22'
-   - '5.20'
-   - '5.18'
-   - '5.16'
    - '5.14'
```

Comment:
It fails because `Mojo::Client::RPC` uses Mojo `6.x` API, meanwhile on tests it fetches Mojo `7.x` API by default. That's why it also fails on `5.14`. So, I'd suggest to turn back all perls check (and may be add 5.24, 5.26, recently released).